lib/mi_bridges/driver/base_page.rb

```diff
@@ -5,10 +5,19 @@ class Driver
     class BasePage
       include Capybara::DSL
 
+      @@run_counts = {}
+
       def initialize(snap_application, logger: nil)
         @logger = logger
         log("filling out page")
         @snap_application = snap_application
+
+        @@run_counts[self.class.to_s] ||= 0
+        if @@run_counts[self.class.to_s] >= 5
```

Comment:
technically, a page could show up 5 times and still be valid (eg: there are more than 5 people in a family, so we have a "Personal Information" page for each one". Thoughts on either looking for a higher number *or* potentially checking for that error text /div on the page?

![screen shot 2017-09-27 at 10 18 27 am](https://user-images.githubusercontent.com/601515/30932364-19393bb4-a37c-11e7-92a5-30c219cad737.png)

Reply:
How many multi-member pages are there where this could pose an issue?  We could "opt out" of those particular pages by resetting the count for that page ... or white-listing them in the base class.

I'm a little iffy on checking for the error div because I'm not sure what to check for that would be universal across them all.

Reply:
Yeah you are right on the "error div" although it might be good to check and see if there is a universal structure, since that seems to be working for the "title" piece of our code (title is always in an h1). 

I think the only multi-member pages are RelationshipInformationPage and PeopleListedPage...so you are thinking we could just not do the looping logic there?